packages/flutter_tools/lib/src/asset.dart

```diff
@@ -18,18 +17,39 @@ import 'flutter_manifest.dart';
 import 'globals.dart' as globals;
 import 'project.dart';
 
-const AssetBundleFactory _kManifestFactory = _ManifestAssetBundleFactory();
-
 const String defaultManifestPath = 'pubspec.yaml';
 
 const String kFontManifestJson = 'FontManifest.json';
 
+/// The effect of adding `uses-material-design: true` to the pubspec is to insert
+/// the following snippet into the asset manifest:
+///
+///```yaml
+/// material:
+/// - family: MaterialIcons
```

Comment:
should line 29 be indented more? Honestly, I never know how to read yaml.

Reply:
Definitely! I think its fixed now...